wagtail_guide/tests/tests.py

```diff
@@ -2,10 +2,7 @@
 from django.contrib.auth.models import Group
 from django.test import TestCase
 from django.urls import reverse
-from wagtail.core.models import Page
-from wagtail.tests.utils import WagtailTestUtils
-from wagtail.tests.utils.form_data import (inline_formset, nested_form_data,
-                                           streamfield)
+from wagtail.test.utils import WagtailTestUtils
```

Comment:
**Questions**: Does this need the `from wagtail import VERSION as WAGTAIL_VERSION` check here

Reply:
Sorry I don't think  I was clear with my question there.

In the upgrade docs: https://docs.wagtail.org/en/stable/releases/3.0.html#changes-to-module-paths doesn't the `from wagtail.test.utils import WagtailTestUtils` need to be conditional for the version?

E.G.
> The wagtail.tests module is renamed to wagtail.test

so we can support v2.16

Reply:
Fixed in https://github.com/kevinhowbrook/wagtailguide/pull/19/commits/db1d83f912809823e1252d3acc89cd89b5085488#diff-0f47fd5fac3c82a47477637399052e15c913d0770c8d1bce16f47994f23dd00e